tests/test_hrrr_preprocessor.py

```diff
@@ -62,6 +68,6 @@ def test_pre_process(self):
         )
 
         self.assertCountEqual(
-            data.keys(),
+            list(data.keys()),
```

Comment:
I just changes this and removed the cast to a list. Doesn't the `keys()` method already return one?

Reply:
Maybe in newer python versions. Python 3.6 returns a `dict_keys` list which is not equal when using `assertListEqual`. But comparing with `assertCountEqual` does work.

Reply:
Think there is not really a need to compare the order and mainly make sure that the keys are present. Hence my earlier change to `CountEquals`

Reply:
Must have been a bad merge on my part then! Looks like we're back to normal.